docs/zh/_sidebar.md

```diff
@@ -1,4 +1,4 @@
-- [首页](/zh/README.md)
+1- [首页](/zh/README.md)
```

Comment:
Please delete this redundent "1" here :)

Reply:
Thks for your review,I have fixed. @seeflood